entry_types/scrolled/package/spec/contentElements/dataWrapperChart/useIframeHeight-spec.js

```diff
@@ -0,0 +1,32 @@
+import {useIframeHeight} from 'contentElements/dataWrapperChart/useIframeHeight';
+
+import {renderHook} from '@testing-library/react-hooks';
+import {fakeParentWindow, tick} from 'support';
+
+describe('useIframeHeight', () => {
+  it('Confirms the default height', async () => {
+    const testURL = 'testUrl/id/1/';
+    fakeParentWindow();
+
+    const {result} = renderHook(() => useIframeHeight(testURL));
+
+    window.postMessage('SOME_MESSAGE', '*');
+    await tick();
+
+    expect(result.current).toEqual('400px');
+  });
+
+  it('removes listener on cleanup', async () => {
+    const testURL = 'testUrl/id/1/';
+    fakeParentWindow();
+    const {unmount} = renderHook(() => useIframeHeight(testURL));
+
+    unmount();
+    window.postMessage('SOME_MESSAGE', '*');
+    await tick();
+
+    const spy = jest.spyOn(window, 'removeEventListener');
```

Comment:
We'd expect `removeEventListener` on `window.document` to be called with parameters 'message' and any `Function`. I think this spec now tests something else